pkgs/servers/sql/materialize/default.nix

```
@@ -20,10 +25,10 @@ let
       inherit hash;
     };
 
-    static = "./src/materialized/src/http/static";
-    cssVendor = "./src/materialized/src/http/static/css/vendor";
-    jsProdVendor = "./src/materialized/src/http/static/js/vendor";
-    jsDevVendor = "./src/materialized/src/http/static-dev/js/vendor";
+    static = "./src/http/static";
+    cssVendor = "./src/http/static/css/vendor";
+    jsProdVendor = "./src/http/static/js/vendor";
+    jsDevVendor = "./src/http/static-dev/js/vendor";
```

These are the correct paths for the npm deps:

```suggestion
    static = "./src/environmentd/src/http/static";
    cssVendor = "./src/environmentd/src/http/static/css/vendor";
    jsProdVendor = "./src/environmentd/src/http/static/js/vendor";
    jsDevVendor = "./src/environmentd/src/http/static-dev/js/vendor";
```